src/attribution/onLCP.ts

```diff
@@ -54,21 +54,21 @@ const attributeLCP = (metric: LCPMetric): LCPMetricWithAttribution => {
               activationStart
           : 0,
       );
-      const lcpResponseEnd = Math.max(
-        lcpRequestStart,
-        lcpResourceEntry ? lcpResourceEntry.responseEnd - activationStart : 0,
-      );
-      const lcpRenderTime = Math.max(
-        lcpResponseEnd,
+      const lcpResponseEnd = Math.min(
+        // Cap at LCP time (videos continue downloading after LCP for example)
         lcpEntry.startTime - activationStart,
+        Math.max(
+          lcpRequestStart,
+          lcpResourceEntry ? lcpResourceEntry.responseEnd - activationStart : 0,
+        ),
       );
 
       attribution = {
         element: getSelector(lcpEntry.element),
         timeToFirstByte: ttfb,
         resourceLoadDelay: lcpRequestStart - ttfb,
         resourceLoadDuration: lcpResponseEnd - lcpRequestStart,
-        elementRenderDelay: lcpRenderTime - lcpResponseEnd,
+        elementRenderDelay: metric.value - lcpResponseEnd,
```

Comment:
I don't remember why I calculated `lcpRenderTime` instead of just using `metric.value`. It's totally possible I just missed that I could, but I'm also wondering if there was another reason...

How much did you check this for edge cases?

Reply:
It was @brendankenny that pointed out this could be `metric.value` above. I can't think of any reason why reason why it shouldn't be that, even with edge cases... 🤔

Reply:
👍